tangram/mapping_utils.py

```diff
@@ -210,6 +258,10 @@ def map_cells_to_space(
         adata_sc = adata_to_cluster_expression(
             adata_sc, cluster_label, scale, add_density=True
         )
+    if mode == "nmf":
+        adata_sc = adata_to_nmf_factor_expression(
+            adata_sc, cluster_label, nmf_coeffs, sample_name, add_density=True
```

Comment:
Does it make sense to assume that you have an "adata_sc"?  For NMF mapping, you're just starting with nmf_factors and nmf_coeffs.  I would recommend passing both of those, and building the adata in the new function.

Reply:
One usually runs `tg.pp_adatas()` before `tg.map_cells_to_space()` (which calls `adata_to_nmf_factor_expression()`). So, I think we can assume that we have `adata_sc`. Is that what you were wondering about?